README.md

```diff
@@ -2,87 +2,100 @@
+| Staging Endpoint | http://council-emissions-calculator-spike-flax.vercel.app/ |
+| Prod Endpoint | https://seechange.codeforaustralia.org/ |
+
+## Table of Contents
+
+- [Project Goals](#project-goals)
+- [System Diagram](#system-diagram)
+- [Getting Started](#getting-started)
+- [Development](#getting-started)
+- [Resourses](#resourses)
+- [Dependencies](#dependencies)
 
 ## Project Goals
 
-- Provide councils a tool to assess their impact on the environment and ...
-- Provide councils with inspirations and next steps to have a more sustainable way of working.
-- Provide councils with inspirations and next steps to engage communities in initiatives that are favorable for the environment.
+- Provide councils a tool to assess their impact on the environment.
+- Provide councils with inspirations and next steps to have a more sustainable way of workingand engage communities in initiatives that are favorable for the environment.
 - Provide a source of information that facilitates the decision process for councils to become more sustainable.
-- to learn more about this project, check out our [project planning](https://docs.google.com/document/d/1h5r1AfbQC8Azy_cdGHGBGOJwSxSqvt0cHKd5MmSd7qo/edit#heading=h.fw4lq2wclkt9) and [trello project board.](https://trello.com/b/ZXaIkclp/civic-makers-2021-l-use-this-one)
+
+To learn more about this project, check out our [project planning](https://docs.google.com/document/d/1h5r1AfbQC8Azy_cdGHGBGOJwSxSqvt0cHKd5MmSd7qo/edit#heading=h.fw4lq2wclkt9) and [trello project board.](https://trello.com/b/iPHuVc0J/civic-makers-2023)
+
+## System Diagram
+
+![Council carbon emission calculator system diagram](https://user-images.githubusercontent.com/88268603/232468816-142513de-c835-4dd3-923f-416635711103.png)
 
 ## Getting Started
 
-<details>
-<summary> First, install dependencies using your preferred package manager... </summary>
+1. Clone the repository to your local machine.
+
+2. Install dependencies using your preferred package manager:
 
 ```
 npm install
-# or
+```
+
+or
+
+```
 yarn install
 ```
 
-Next, run the development server:
+3. Run the development server:
 
 ```
 npm run dev
-# or
+```
+
+or
+
+```
 yarn dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+4. Open [http://localhost:3000](http://localhost:3000) in your browser to view the carbon emission survey website locally.
+
+## Development
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+We have 2 endpoints: [Staging](http://council-emissions-calculator-spike-flax.vercel.app/) and [Production](https://seechange.codeforaustralia.org/).
 
-</details>
+1. Create a new branch locally. _No need to fork the project, you can create a branch in the original one._
+2. Once you make a code change, commit and publish your branch.
+3. Raise a PR against our [Staging Endpoint](http://council-emissions-calculator-spike-flax.vercel.app/). 
```

Comment:
> 1. Create a new branch locally. _No need to fork the project, you can create a branch in the original one._

This statement is true, but it only applies to people who are listed as members in this repo. New contributors, who are not yet members of the repo, **will need to fork** this repo and then contribute their changes via the fork.

How about this:

```suggestion
Please refer to [this doc](https://docs.github.com/en/get-started/quickstart/fork-a-repo) for instructions on how to fork this repo and contributing your changes.

**N.B:** Please note well that the *default branch* for this repo is the **staging** branch, not the *main* branch; Any new pull request raised for this repo will have its changes pushed to **staging** branch by default when the PR is approved and merged.

The **staging** branch is hosted on our [Staging endpoint](http://council-emissions-calculator-spike-flax.vercel.app/). 

The **main** branch is hosted on our [Production endpoint](https://seechange.codeforaustralia.org/).

```